guides/main_concepts/documents.md

```diff
@@ -28,24 +32,26 @@ curl -X POST `http://localhost:7700/indexes/movies/documents` \
 ]'
 ```
 
-A **document must contain** [the primary key field](/guides/main_concepts/documents.md#primary-key) to be indexed in MeiliSearch.
+In order to be indexed, each **document must contain** [the primary key field](/guides/main_concepts/documents.md#primary-key).
 
 ## Fields
 
-All fields present in a document are automatically <clientGlossary word="searchable" /> and <clientGlossary word="displayed" />.
-The way MeiliSearch handles a field is customizable in the settings <Badge text="soon" type="warn"/>. You can make a field only searchable, or only displayed, or none, in this case, MeiliSearch will completely ignore the field when it is sent on document addition.
+By default, all fields included in a document are <clientGlossary word="searchable" /> and automatically <clientGlossary word="displayed" />.
+You can adjust how a field get handled by MeiliSearch in the settings <Badge text="soon" type="warn"/>. It can either be searchable or displayed, or none of both. In the latter case, the field will be completely ignored when a document is sent.
```

Comment:
```suggestion
You can adjust how a field get handled by MeiliSearch in the settings <Badge text="soon" type="warn"/>. It can either be searchable or displayed, both, or none of both. In the latter case, the field will be completely ignored when a document is sent.
```